src/main/java/openfasttrack/importer/markdown/MarkdownImporter.java

```diff
@@ -66,7 +68,8 @@ public void runImport()
         }
         catch (final IOException exception)
         {
-            LOG.warning("IO exception after line " + lineNumber);
+            LOG.warning(
+                    "IO exception after line " + lineNumber + " of file '" + this.fileName + "'");
```

Comment:
I would prefer all input related warnings to be in a uniform, parseable format (e.g. Java stack-trace-style). in the best case that allows editors like Eclipse to jump to the file in question.